tokio/tests/support/mpsc_stream.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(dead_code)]
+
```

Comment:
This isn't dead?

Reply:
It may be that not everything is used in each test. Since each test file is a separate target, it may raise warnings.

Reply:
RA complains, its in test, going to keep this for now.